build/ts/app.ts

```diff
@@ -215,8 +236,14 @@ class Application {
         .empty();
 
       for (const sticker of card.stickers) {
+        const stickerClass = sticker.Class !== '' ? ` sticker-${sticker.Class}` : '';
```

Comment:
Admittedly I'm not a front-end person but this `sticker-` prefix feels weird—can we just have a `sticker` class and then a bunch of classes like `team-a`, `comments-to-resolve` etc?

I think this change might make this implementation a bit tidier too.

Reply:
Well, I think that is rather complicated. After researching, we will move from a simple `.card:has(.sticker-team-a)` to a multiline implementation in a loop in javascript which is fairly difficult to read. We would have to iterate across each card, find the child, and find if one has 2 the classes.

Maybe I am wrong, but If the goal is "implementation a bit tidier too", that would be the opposite.

Reply:
I also think it would be a very bad idea. The reason is that the stickers are coming from the input of the user, from the config file `stickers.yml`. If somebody creates a sticker called `card`, for instance, all the implementation would blowup

I think it is safer to put a prefix to it.